docs/csharp/language-reference/operators/member-access-operators.md

```diff
@@ -147,9 +147,23 @@ If `a.x` or `a[x]` is of a non-nullable value type `T`, `a?.x` or `a?[x]` is of
 In the preceding example, if you don't use the `??` operator, `numbers?.Length < 2` evaluates to `false` when `numbers` is `null`.
 
 > [!NOTE]
-> The `?.` operator evaluates its left-hand operand no more than once, guaranteeing that it cannot be changed to `null` after being verified as non-null.
+> The `?.` operator evaluates its left-hand operand no more than once, guaranteeing that it can't be changed to `null` after being verified as non-null.
 
-The null-conditional member access operator `?.` is also known as the Elvis operator.
+Beginning in C# 14, assignment is permissible with a null conditional access expression (`?.` and `?[]`) on reference types. For example, see the following method:
+
+:::code language="csharp" source="snippets/shared/NullCoalescingOperator.cs" id="NullForgivingAssignment":::
+
+The preceding example shows assignment to a property and an indexed element on a reference type that might be null. An important behavior for this assignment is that the expression on the right hand side of the `=` is evaluated only once the left hand side is known to be non-null. For example, in the following code, the function `GenerateNextIndex` is call only when the `values` array isn't null. If the `values` array is null, `GenerateNextIndex` isn't called:
+
+:::code language="csharp" source="snippets/shared/NullCoalescingOperator.cs" id="NullForgivingAssignment":::
+
+In other words, the preceding code is equivalent to the following code using an `if` statement for the null check:
+
+:::code language="csharp" source="snippets/shared/NullCoalescingOperator.cs" id="EquivalentIfStatement":::
+
+In addition to assignment, any form of [compound assignment](./assignment-operator.md#compound-assignment), such as `+=` or `-=` are allowed. However, increment (`++`) and decrement (`--`) aren't allowed.
+
+This enhancement doesn't classify a null conditional expression as a variable. It can't be `ref` assigned, nor can it be assigned to a `ref` variable, or passed to a method as a `ref` or `out` argument.
```

Comment:
```suggestion
This enhancement doesn't classify a null conditional expression as a variable. It can't be `ref` assigned, nor can it be assigned to a `ref` variable or passed to a method as a `ref` or `out` argument.
```